src/components/pid-component/pid-component.tsx

```diff
@@ -257,7 +257,7 @@ export class PidComponent {
               <span
                 class={
                   this.currentLevelOfSubcomponents === 0
-                    ? 'group rounded-md shadow-md border text-clip inline-flex flex-grow py-0.5 px-1 open:align-top open:w-full ease-in-out transition-all duration-200 overflow-y-hidden font-bold font-mono cursor-pointer list-none bg-white overflow-x-hidden space-x-3 flex-nowrap flex-shrink-0 items-center'
+                    ? 'group text-clip inline-flex flex-grow py-0.5 px-1 open:align-top open:w-full ease-in-out transition-all duration-200 overflow-y-hidden font-bold font-mono cursor-pointer list-none overflow-x-hidden space-x-3 flex-nowrap flex-shrink-0 items-center'
```

Comment:
Maybe some elevation from the background would be nice. It doesn't have to be a bright white background, but you could decrease the opacity of the background (e.g. bg-white/50) https://tailwindcss.com/docs/background-color. Additionally, I'd like some kind of rounded border around the component to make it more visible. If you're looking for a possibility to save space, you could decrease or inherit the font size.